<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VillageDialog</class>
 <widget class="QDialog" name="VillageDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>225</width>
    <height>155</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Village</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Village name:</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="DialogBtn">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="VillageInput">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Adding to:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>-- World --</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>DialogBtn</sender>
   <signal>accepted()</signal>
   <receiver>VillageDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>83</x>
     <y>88</y>
    </hint>
    <hint type="destinationlabel">
     <x>107</x>
     <y>64</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>DialogBtn</sender>
   <signal>rejected()</signal>
   <receiver>VillageDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>149</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel">
     <x>46</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
